engine/src/main/java/com/redhat/cloud/notifications/processors/SystemEndpointTypeProcessor.java

```diff
@@ -75,10 +81,10 @@ public Set<com.redhat.cloud.notifications.processors.email.connector.dto.Recipie
     protected List<String> getSubscribers(final Event event, final SubscriptionType subscriptionType) {
         final EventType eventType = event.getEventType();
 
-        return this.subscriptionRepository.getSubscribersByEventType(
-            event.getOrgId(),
-            eventType.getId(),
-            subscriptionType
-        );
+        if (subscriptionType.isSubscribedByDefault()) {
+            return subscriptionRepository.getUnsubscribers(event.getOrgId(), eventType.getId(), subscriptionType);
+        } else {
+            return subscriptionRepository.getSubscribers(event.getOrgId(), eventType.getId(), subscriptionType);
+        }
     }
 }
```

Comment:
If you look at the old code, when this method was executed for the drawer (which is subscribed by default), the records we retrieved from the `email_subscriptions` table we unsubscriptions (`subscribed = false`). Now, things a bit less confusing but the behavior should be the same than the old one.

Reply:
We retrieved these records:
https://github.com/RedHatInsights/notifications-backend/blob/922fb35ec38b3f9dff439b29ccb630e65ce7d310/backend/src/main/java/com/redhat/cloud/notifications/db/repositories/SubscriptionRepository.java#L60

Reply:
Before this PR, the email_subscriptions table looked like this:
| | Email | Drawer |
| - | - | - |
| Subscription | Record with subscribed = true | No record |
| Unsubscription | No record | Record with subscribed = false |

After this PR, it will look like this:
| | Email | Drawer |
| - | - | - |
| Subscription | Record with subscribed = true | Record with subscribed = true |
| Unsubscription | Record with subscribed = false | Record with subscribed = false |

Reply:
All right, thank you very much for the detailed explanation!